boards/arm64/fvp_baser_aemv8r/fvp_baser_aemv8r.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 Arm Limited (or its affiliates). All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: fvp_baser_aemv8r
+name: FVP Emulation FVP_BaseR_AEMv8R
+arch: arm64
+type: sim
+toolchain:
+  - zephyr
+  - cross-compile
+ram: 128
```

Comment:
I'm not sure this tag actually makes a difference here, but it isn't in sync with the value (2048) in the devicetree above.

Reply:
This tag seems to make no difference, but anyway I'll fix that. Thanks so much